### DISCUSSION TITLE
account blocked

### DISCUSSION THREAD

**Discussion by aistudylearn0** [vercel/community#1930] [Help] [answered]
### Summary

I just register by my github account , then login get response "This user account is blocked. Contact registration@vercel.com for more information."  I don't know why, i can't open the link "registration@vercel.com". Help needed!!!

**Comment by swarnava** [comment, marked as answer]
Hi. Please contact to the given email address to unblock your account.

**Reply by haoguangxie** [reply]
我也遇到这个问题了，账户：haoguangxie9@gmail.com 
麻烦可以帮我解锁这个账户么？谢谢
